packages/pilot/src/pages/Transactions/Search/Search.js

```diff
@@ -85,11 +85,19 @@ const normalizeQueryDatesToString = pipe(
 
 const stringToMoment = str => moment(str)
 
+const normalizeDateTime = property => (date) => {
+  if (property === 'start') {
+    return date.startOf('day')
+  }
+
+  return date.endOf('day')
+}
+
 const normalizeStringToDate = property => pipe(
   prop(property),
   unless(
     either(isNil, isEmpty),
-    pipe(stringToMoment, objOf(property))
+    pipe(stringToMoment, normalizeDateTime(property), objOf(property))
```

Comment:
sugestão de formatar esse pipe pra ficar mais legivel 
```suggestion
    pipe(
      stringToMoment,
      normalizeDateTime(property),
      objOf(property)
    )
```